app/enterprise/0.33-x/developer-portal/best-practice.md

```diff
@@ -0,0 +1,204 @@
+│   │   │   └── toaster.js
+│   │   ├── pages
+│   │   │   └── dashboard.vue
+│   │   └── schemas
+│   │       ├── ACL.js
+│   │       ├── BasicAuth.js
+│   │       ├── HMAC.js
+│   │       ├── JWT.js
+│   │       ├── KeyAuth.js
+│   │       ├── OAuth2.js
+│   │       └── plugins.js
+│   ├── dashboard.js
+│   ├── site.js
+│   └── utils
+│       ├── common.js
+│       ├── error.js
+│       ├── route.js
+│       ├── spec.js
+│       └── vitals.js
+└── templates
+    ├── pages
+    │   ├── 404.hbs
+    │   ├── about.hbs
+    │   ├── dashboard.hbs
+    │   ├── documentation
+    │   │   ├── api1.hbs
+    │   │   ├── api2.hbs
+    │   │   └── loader.hbs
+    │   ├── guides
+    │   │   ├── 5-minute-quickstart.hbs
+    │   │   ├── index.hbs
+    │   │   ├── kong-architecture-overview.hbs
+    │   │   ├── kong-ee-introduction.hbs
+    │   │   ├── kong-vitals.hbs
+    │   │   └── uploading-spec.hbs
+    │   ├── index.hbs
+    │   ├── unauthenticated
+    │   │   ├── 404.hbs
+    │   │   ├── index.hbs
+    │   │   ├── login.hbs
+    │   │   ├── register.hbs
+    │   │   └── unauthorized.hbs
+    │   └── user.hbs
+    ├── partials
+    │   ├── custom-js.hbs
+    │   ├── footer.hbs
+    │   ├── header.hbs
+    │   ├── kong-docs-sidebar.hbs
+    │   ├── layout.hbs
+    │   ├── sidebar-spec.hbs
+    │   ├── sidebar.hbs
+    │   ├── spec-dropdown.hbs
+    │   ├── spec-renderer.hbs
+    │   └── unauthenticated
+    │       ├── auth-js.hbs
+    │       ├── code-snippet-languages.hbs
+    │       ├── custom-css.hbs
+    │       ├── footer.hbs
+    │       ├── header.hbs
+    │       ├── layout.hbs
+    │       ├── login-actions.hbs
+    │       └── title.hbs
+    └── specs
+        ├── admin.json
+        ├── files.yaml
+        └── vitals.yaml
+```
+
+When the file structure is setup, you can make use of [GitHub Webhooks](https://developer.github.com/webhooks/creating/) and every time you merge a change into the Master branch, it triggers a push of files to the portal. This allows your developers to work in an environment they are familiar with, and maintain a history of changes to your portal. It also means you can easily roll back to a previous version.
+
+## <a name=partial></a>Using Effective Partial Usage
+
+Partials in the portal allow you to create reusable elements of a page. A page will traditionally be made up of a group of partials. For example, if you wanted to create a page announcing a new product, you would probably want to include things such as the header, footer, sidebar etc. Each of those elements are saved as partials. For more information about partials, please refer to [our documentation](https://docs.konghq.com/enterprise/0.33-x/developer-portal/file-management/#partials).
+
+partials allow you to control what is on every page.n In order for your portal to load quickly, it is vital to keep these files as small as possible. The general rule is partials should be around 500 kB. They should never be larger than 1 MB.
```

Comment:
There is some incorrect capitalization and a typo here.

Reply:
Thanks!